CONTRIBUTING.md

```diff
@@ -12,28 +12,25 @@ Please follow these steps to report such a problem
 1. [Create an issue](https://github.com/octokit/routes/issues/new). Follow the instructions in the `<!-- comments -->`
 2. If you would like to work resolving this issue, let us know :)
    Otherwise you are done here, thank you!
-3. Fork the repository. Clone it to your computer and run the tests to make sure
-   that everything works as expected.
-4. Edit the `*.json` file to to what is expected from the issue
+3. Fork the repository. Clone it to your computer and run the tests to make sure that everything works as expected.
+4. Edit the `openapi/[api]/operations/[scope]/[operationId].json` file to what is expected from the issue
 5. Run the tests, they should fail now
 6. Commit the change with `test: ...` and start a pull request.
-7. Let us know if you’d like to work on resolving the failing test. Otherwise
-   you are done here, thank you, this is really great!
-8. Figure out how to make your failing test pass. You can run a test for only
-   `*.json` file you edited based on the documentation URL by running e.g.
+7. Let us know if you’d like to work on resolving the failing test.
+   Otherwise you are done here, thank you, this is really great!
+8. Figure out how to make your failing test pass. You can run a test for just the operation you edited, based on its documentation URL, by running:
 
-   ```
-   TEST_URL=https://developer.github.com/v3/repos/#get ./node_modules/.bin/tap test/integration/single-endpoint-test.js
+   ```bash
+   $ TEST_URL=https://developer.github.com/v3/repos/#get ./node_modules/.bin/tap test/integration/endpoints-test.js
```

Comment:
I usually don’t add the `$` because it makes it harder to copy & paste. Is it needed for better syntax highlighting?

Reply:
pushed up a change that removes them...i do that in documentation so the context is instantly recognizable; for readability, especially since docs are often just scanned instead of read, but no big deal